velox/docs/functions/spark/array.rst

```diff
@@ -196,6 +196,25 @@ Array Functions
         SELECT size(NULL, true); -- -1
         SELECT size(NULL, false); -- NULL
 
+.. spark:function:: slice(array(E), start, length) -> array(E)
+
+    Returns a subarray starting at 1-based index ``start`` or from end if negative,
+    with ``length`` elements.
+
+        Return empty array if ``start`` point outside of the array;
```

Comment:
Would you generate the docs and make sure they render nicely?

Reply:
The generated doc is

![Screenshot 2024-07-23 at 11 15 53](https://github.com/user-attachments/assets/5f7c6668-515f-4904-8903-3759cfe0e8dc)